magefiles/magefile.go

```diff
@@ -196,7 +199,8 @@ func (CI) setRequiredEnvVars() error {
 			os.Setenv("E2E_TEST_SUITE_LABEL", testSuiteLabel)
 
 		} else if openshiftJobSpec.Refs.Repo == "infra-deployments" {
-			os.Setenv("INFRA_DEPLOYMENTS_ORG", pr.RemoteName)
+
+			os.Setenv("INFRA_DEPLOYMENTS_ORG", pr.Organization)
```

Comment:
Not sure if this change is related to the PR. Also `pr.Organization` wouldn't work in this case, since it points to the upstream org, i.e. currently `redhat-appstudio`. see [this PR](https://github.com/redhat-appstudio/e2e-tests/pull/249) for more info

Reply:
Yeah, I got confused here when vscode was showing me the changes when I rebased the latest changes locally. I'll figure out how to undo this change.